extras/macosLauncher/starter.cpp

```diff
@@ -1,11 +1,33 @@
+// Copyright (c) 2005 - 2019 Settlers Freaks (sf-team at siedler25.org)
+//
+// This file is part of Return To The Roots.
+//
+// Return To The Roots is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 2 of the License, or
+// (at your option) any later version.
+//
+// Return To The Roots is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Return To The Roots. If not, see <http://www.gnu.org/licenses/>.
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+
 #include "RttrConfig.h"
+
```

Comment:
Don't use newlines. I was already hunting for some of them to remove them. Use clang-format to order includes

Reply:
sure I will use newlines. thats what newlines are for. this way you properly see which is local/module, project and system header.

Reply:
Although I've seen this used before, but don't like it. It is not really required as namespacing and naming should make things clear (`"local.h", <module/h.h>, <boost/h.h>, <system>`). Besides that I don't understand the benefit in seeing that distinction. Why would it be important WHERE a header lives (except for ordering which is done by clang-format) On the other hand I find scrolling annoying and hence try to reduce vertical space. And again you will need to painfully manually check that newlines are consistently used.